src/Connection.php

```diff
@@ -5,24 +5,35 @@
 use Illuminate\Database\Connection as BaseConnection;
 use Illuminate\Support\Arr;
 use InvalidArgumentException;
+use Jenssegers\Mongodb\Concerns\TransactionManager;
 use MongoDB\Client;
+use MongoDB\Database;
+use MongoDB\Driver\Session;
 
 class Connection extends BaseConnection
 {
+    use TransactionManager;
+
     /**
      * The MongoDB database handler.
      *
-     * @var \MongoDB\Database
+     * @var Database
      */
     protected $db;
 
     /**
      * The MongoDB connection handler.
      *
-     * @var \MongoDB\Client
+     * @var Client
      */
     protected $connection;
 
+    /**
+     * A list of transaction session.
+     * @var Session
```

Comment:
I realize we can't change the types of `$db` and `$connection` without breaking BC, but we should be able to apply a type to `$session` since it's being introduced here. This may relate to my earlier comment on the trait's `getSession()` method.

Reply:
Done